README.md

```diff
@@ -1,2 +1,17 @@
-# Name-subject-to-change
-{NAME} - A Distributed VCS in Python. A distributed revision control system in Python. an Open Source project covered by the GNU General Public License.
+# TIT
+{TIT} - A Distributed VCS in Python. A distributed revision control system in Python. an Open Source project covered by the GNU General Public License.
```

Comment:
this should be like this

TIT - A Distributed Version Control System in Python. an Open Source project covered by the GNU General Public License.

Reply:
okay sure 

I will edit this